joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitorHelpers.scala

```diff
@@ -288,7 +289,8 @@ trait PythonAstVisitorHelpers { this: PythonAstVisitor =>
         .map { case (keyword: String, argNode) => keyword + " = " + codeOf(argNode) }
         .mkString(", ") +
       ")"
-    val callNode = nodeBuilder.callNode(code, name, DispatchTypes.DYNAMIC_DISPATCH, lineAndColumn)
+    val passedMethodFullName = if (methodFullName == "") name else methodFullName + name
```

Comment:
If you use `Option` types, then this can become:
```scala
val passedMethodFullName = methodFullName match {
  case Some(fullName) => fullName + name
  case None => name
}
```